doc/source/api_changes.rst

```diff
@@ -23,6 +23,7 @@ Version 0.14
   favor of ``skimage.restoration.denoise_nl_means``.
 - ``skimage.measure.LineModel`` has been removed in favor of
   ``skimage.measure.LineModelND``.
+- In ``skimage.feature.hog`` visualise has been changed to visualize.
```

Comment:
Also add an entry in `TODO.txt` to remind us to remove deprecated keyword in 0.16